resources.management_groups.tf

```diff
@@ -83,6 +83,7 @@ resource "azapi_resource" "diag_settings" {
   type                      = "Microsoft.Insights/diagnosticSettings@2021-05-01-preview"
   name                      = "toLA"
   parent_id                 = each.key
+  location                  = "global"
```

Comment:
This is what the Azure API returns, we need to have something in there even though we are ignoring changes for this resource - otherwise the lifecycle argument errors. It has been a perennial pain this one but this configuration is idempotent and works.